### DISCUSSION TITLE
Confirm exit on esc

### DISCUSSION THREAD

**Discussion by Vantropix** [ppy/osu#17474] [Ideas]
An option that lets you toggle if you need to double press esc to pause
This is from other games like stepmania and switching from games makes me personally get into a really annoying situation. Instead of pausing normally i'll pause then immediately unpause, with the cooldown of pausing again a failiure is near guaranteed so this would be a way to avoid that situation and bring some familliarity to the game from other games for an easier time switching games.

**Comment by bdach** [comment]
by default pressing escape doesn't pause, you have to hold it. did you set hold activation time to 0?

**Reply by Vantropix** [reply]
Yes, since the same situation happens with holding enabled, i forget to hold and instead double tap which still will have me fail
And since i tap esc about the same amount of time both times having a short hold time wouldn't do much since i would still unpause

**Comment by peppy** [comment]
I don't think we'll add an option for double tap. You would need to choose between hold or single tap.
